src/rgw/Makefile.am

```diff
@@ -160,7 +160,7 @@ radosgw_SOURCES = \
 	rgw/rgw_main.cc
 
 radosgw_CFLAGS = -I$(srcdir)/civetweb/include -fPIC -I$(srcdir)/xxHash
-radosgw_CXXFLAGS = ${RGW_CXXFLAGS} ${AM_CXXFLAGS}
+radosgw_CXXFLAGS = ${RGW_CXXFLAGS} ${AM_CXXFLAGS} -I$(srcdir)/Beast/include
```

Comment:
i think we want this in `librgw_la_CXXFLAGS`. the Beast types could be useful outside of the asio frontend